### PR TITLE
Feature/7400 speedup q3 fundings search and link 2

### PR DESCRIPTION
**Pull request by DanielPalafox**

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[[speedup Q3] Fundings: search and link (2)](https://trello.com/c/ze4D7fpp/7400-speedup-q3-fundings-search-and-link-2)

File: src/app/record/components/work-stack-group/modals/modal-works-search-link.component.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      </div>
+      <mat-form-field
+        appearance="outline"
+        class="mat-form-field-min search-link-select"
+      >
+        <mat-select
+          [(value)]="workTypeSelected"
+          (selectionChange)="searchAndLink()"
+        >
+          <mat-option value="All">All</mat-option>
```
Comment (leomendoza123) [inline src/app/record/components/work-stack-group/modals/modal-works-search-link.component.html:41]: Require the i18n tag

Comment (leomendoza123) [reply src/app/record/components/work-stack-group/modals/modal-works-search-link.component.html:41]: also the next ALL on the other mat-form-field

File: src/app/record/components/work-stack-group/work-stack-group.component.ts [hunk 2]
```diff
@@ -50,6 +51,9 @@ export class WorkStackGroupComponent implements OnInit {
   @Input() expandedContent: boolean
   @Output() total: EventEmitter<any> = new EventEmitter()
   @Output() expanded: EventEmitter<any> = new EventEmitter()
+
+  modalWorksSearchLinkComponent = ModalWorksSearchLinkComponent
```
Comment (leomendoza123) [inline src/app/record/components/work-stack-group/work-stack-group.component.ts:55]: Seems like never use

File: src/app/record/components/funding-stacks-groups/modals/modal-funding-search-link/modal-funding-search-link.component.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+  loadFundingImportWizardList(): void {
+    this._recordFundingsService
+      .loadFundingImportWizardList()
+      .pipe(takeUntil(this.$destroy))
+      .subscribe(
+        (recordImportWizards) => {
+          this.recordImportWizardsOriginal = recordImportWizards
+          this.recordImportWizards = this.recordImportWizardsOriginal
+          // recordImportWizards.forEach((recordImportWizard) => {
```
Comment (leomendoza123) [inline src/app/record/components/funding-stacks-groups/modals/modal-funding-search-link/modal-funding-search-link.component.ts:38]: Clean unrequired comments

**Review by leomendoza123 — CHANGES_REQUESTED** [review]

File: src/app/record/components/work-stack-group/work-stack-group.component.ts (hunk 2 above)
Comment (DanielPalafox) [reply src/app/record/components/work-stack-group/work-stack-group.component.ts:55]: I will remove it

File: src/app/record/components/funding-stacks-groups/modals/modal-funding-search-link/modal-funding-search-link.component.ts (hunk 3 above)
Comment (DanielPalafox) [reply src/app/record/components/funding-stacks-groups/modals/modal-funding-search-link/modal-funding-search-link.component.ts:38]: Done!

File: src/app/record/components/work-stack-group/modals/modal-works-search-link.component.html (hunk 1 above)
Comment (DanielPalafox) [reply src/app/record/components/work-stack-group/modals/modal-works-search-link.component.html:41]: Done!

**Review by leomendoza123 — APPROVED** [review]

**Pull request merged**